app/locales/en/translations.coffee

```diff
@@ -259,6 +259,7 @@ I18nTranslationsEn =
       "invalid_inventory": "Inventory number is invalid."
       "invalid_quantity": "Quantity can not be blank."
       "invalid_description": "Description can not be blank."
+      "invalid_location": "Location can not be blank"
```

Comment:
@viveksriram1 , add full-stop at the end of the sentence.

Reply:
Sure @Darshan-Chauhan